docs/pages/api/index.md

```diff
@@ -12,33 +12,115 @@ env = gym.make('CartPole-v0')
+>>
+>> This method returns a tuple `(observation, reward, done, info)`:
+>>- `observation` (**object**): agent's observation of the current environment.
+>> This may, for instance, be a numpy array containing the positions and velocities of certain objects.
+>>- `reward` (**float**) : amount of reward returned after previous action
+>>- `done` (**bool**): whether the episode has ended, in which case further `step()` calls will return undefined results.
+>> A done signal may be emitted for different reasons: Maybe the task underlying the environment was solved successfully, 
+>> a certain timelimit was exceeded, or the physics simulation has entered an invalid state. `info` may contain additional
+>> information regarding the reasone for a done signal.
+>>- `info` (**dict**): contains auxiliary diagnostic information (helpful for debugging, and sometimes learning)
+
+>### Resetting
+>`reset(*, seed: Optional[int] = None, return_info: bool = False, options: Optional[dict] = None,) -> Union[ObsType, tuple[ObsType, dict]]`
+>
+>Resets the environment to an initial state and returns an initial observation.
+>>**Parameters**
+>>- `seed`(**int** or **None**): The seed that is used to initialize the environments PRNG. If the environment does not already
+>> have a PRNG and `seed=None`(the default option) is passed, a seed will be chosen from some source of entropy (e.g. timestamp 
+or /dev/urandom). However, if the environment already has a PRNG and `seed=None` is pased, the PRNG will *not* be reset. 
+If you pass an integer, the PRNG will be reset even if it already exists. Usually, you want to pass an integer *right after the environment
+has been initialized and then never again*. Please refer to the minimal example above to see this paradigm in action.
+>>- `return_info`(**bool**): If true, return additional information along with initial observation. This info should be analogous
+to the info returned in `step`
+>>- `options`(**dict** or **None**):
+>
+>>**Returns**
+>>
+>>This method will return an observation of the initial environment state. If `return_info=True` is passed, the method returns a tuple
+>> `(observation, info)`, where `observation` is the actual observation and `info` is some auxiliary information complementing `observation`.
+>> The dictionary `info` should be analogous to the `info` returned by `step`. If `return_info=False` is passed, only `observation` is returned.
+
+>### Rendering
+> 
+>`render(mode: str="human") -> Optional[Union[numpy.ndarray, str]]`
+>>**Parameters**
+>>- `mode`(**str**): This parameter specifies how the environment is rendered. The set of supported modes varies per environment. (And some
+environments do not support rendering at all.) By convention, if mode is:
+>>      - "human": render to the current display or terminal and return nothing. Usually for human consumption.
+>>      - "rgb_array": Return an numpy.ndarray with shape (x, y, 3), representing RGB values for an x-by-y pixel image, suitable
```

Comment:
np.ndarray

And maybe change `x, y` to `width, height`. But check which one is which. I'm frankly not sure.

Reply:
(this will also remove the need for the `x-by-y pixel image` part